custom_dc/load_data.sh

```diff
@@ -0,0 +1,319 @@
+#!/bin/bash
+# Script to process stats using simple stats loader.
+# Defaults
+MODE="customdc"
+OUTPUT_DIR=".data/output_$MODE"
+USAGE="Script to load data into custom dc using simple importer.
+Usage: $(basename $0) [Options]
+Options:
+  -c <file>       Json config file for stats importer
+  -e <file>       Load environment variables from file.
+  -i <dir>        Input directory to process
+  -k <api-key>    DataCommons API Key
+  -o <dir>        Output folder for stats importer. Default: $OUTPUT_DIR
+  -m <customdc|maindc> Mode of operation for simple importer. Default: $MODE
+  -j <jar>        DC Import java jar file.
+                    Download latest from https://github.com/datacommonsorg/import/releases/
+  -r <steps>      Steps to run. Can be one or more from:
+                    stats, validate
+  -s <cloud_sql>  Cloud SQL instance. Also set DB_USER and DB_PASS with -u, -p
```

Comment:
Let's get all of these from the env.list file

Reply:
If these are set in env.list they will apply and no need to set in commandline.

Commandline option is just to override settings in env.list that can be useful when testing.

Reply:
I see. So then adjust the documentation to say that they are optional, only needed to override what's in the env.list file.

Reply:
Removed these options from the help string. Still support commandline options for internal use.